### PR TITLE
add softsign, softplus and unittests

### PR DESCRIPTION
**Pull request by simanw304**
in:
math.scala
PythonAutoGrad.scala
autograd.py
test_operator.py

### CODE REVIEW

File: pyzoo/zoo/pipeline/api/autograd.py [hunk 1]
```diff
@@ -68,6 +68,14 @@ def epsilon():
     return Variable.from_jvalue(callBigDlFunc("float", "epsilon"))
 
 
+def softsign(a):
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/autograd.py]: add some doc ?

**Comment by zhichao-li** [comment]
It looks good in general except that some documents are missed for the new added operator.

**Comment by zhichao-li** [comment]
Let's merge this once pass the Jenkins test.

**Comment by simanw304** [comment]
jenkins passed http://172.168.2.101:8080/job/ZOO-PR-Validation/528/ 
@zhichao-li

**Pull request merged**